test/dymaptic.GeoBlazor.Core.Test/dymaptic.GeoBlazor.Core.Test.csproj

```diff
@@ -19,7 +19,6 @@
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="SixLabors.ImageSharp" Version="3.0.2"/>
```

Comment:
do we know where this came from, since it showed up in a number of repos?

Reply:
I added it to the unit tests when trying to implement a "snapshot comparison" Playwright test long ago. The test was always shaky and didn't work out.

It is used for handling attachments in some of our other apps.